tests/integration/cli/test_help.py

```diff
@@ -13,7 +13,7 @@ def test_help_page_for_non_aliased_actions():
         "API Documentation: https://www.linode.com/docs/api/linode-instances/#linodes-list"
         in output
     )
-    assert "You may filter results with:" in output
+    assert "wrong assertion" in output
```

Comment:
Is this an intentional change?

Reply:
No I thought I reverted it back. Should be fixed now